docs/outputs.rst

```diff
@@ -366,6 +376,67 @@ For CompCor decompositions, entries include:
     Entries that are not saved in the data file for denoising are still stored in metadata with the
     ``dropped`` prefix.
 
+.. caution::
+    Only a subset of these CompCor decompositions should be used for further denoising.
+    The original Behzadi aCompCor implementation [Behzadi2007]_ can be applied using
+    components from the combined masks, while the more recent Muschelli implementation
+    [Muschelli2014]_ can be applied using
+    the :abbr:`WM (white matter)` and :abbr:`CSF (cerebro-spinal fluid)` masks.
+    To determine the provenance of each component, consult the metadata file (described above).
+
+    There are many valid ways of selecting CompCor components for further denoising.
+    In general, the components with the largest singular values (i.e., those that
+    explain the largest fraction of variance in the data) should be selected.
+    *fMRIPrep* outputs components in descending order of singular value.
+    Common approaches include selecting a fixed number of components (e.g., the
+    first 5 or 6), using a quantitative or qualitative criterion (e.g., elbow, broken
+    stick, or condition number), or using sufficiently many components that a minimum
+    cumulative fraction of variance is explained (e.g., 50%).
+
+.. caution::
+    Similarly, if you are using anatomical or temporal CompCor it may not make sense
+    to use the ``csf``, or ``white_matter`` global regressors -
+    see `#1049 <https://github.com/poldracklab/fmriprep/issues/1049>`_.
+    Conversely, using the overall ``global_signal`` confound in addition to CompCor's
+    regressors can be beneficial (see [Parkes2018]_).
+
+.. danger::
+    *fMRIPrep* does high-pass filtering before running anatomical or temporal CompCor.
+    Therefore, when using CompCor regressors, the corresponding ``cosine_XX`` regressors
+    should also be included in the design matrix.
+
+.. admonition:: See also
+
+    This didactic `discussion on NeuroStars.org
+    <https://neurostars.org/t/fmrirep-outputs-very-high-number-of-acompcors-up-to-1000/5451>`__
+    where Patrick Sadil gets into details about PCA and how that base technique applies
+    to CompCor in general and *fMRIPrep*'s implementation in particular.
+
+**AROMA confounds**.
+:abbr:`AROMA (Automatic Removal Of Motion Artifacts)` is an :abbr:`ICA (independent components analysis)`
+based procedure to identify confounding time series related to head-motion [Prium2015]_.
+ICA-AROMA can be enabled with the flag ``--use-aroma``.
+
+- ``aroma_motion_XX`` - the motion-related components identified by ICA-AROMA.
+
+.. danger::
+    If you are already using ICA-AROMA's cleaned-data (``~desc-smoothAROMAnonaggr_bold.nii.gz``),
```

Comment:
"ICA-AROMA's cleaned-data" is weird. Both the `'s` and the hyphen between cleaned and data. You could do

```suggestion
    If you are already using ICA-AROMA-cleaned data (``~desc-smoothAROMAnonaggr_bold.nii.gz``),
```

Or perhaps

```suggestion
    If you are already using AROMA-cleaned data (``~desc-smoothAROMAnonaggr_bold.nii.gz``),
```